lms/static/js/learner_dashboard/views/program_details_sidebar_view.js

```diff
@@ -81,6 +91,17 @@ class ProgramDetailsSidebarView extends Backbone.View {
 
     return type.replace(/\s+/g, '-');
   }
+
+  trackPathwayClicked(event) {
+    var button = event.currentTarget;
+    window.analytics.track('edx.bi.dashboard.program.pathway.clicked', {
+      category: 'pathways',
+      program_uuid: this.programModel.attributes.uuid,
+      program_name: this.programModel.attributes.title,
```

Comment:
Program and pathway names are interesting. Those might change over time, unlike the uuid (one hopes). Might be harder to match events over time using those.

Reply:
Marco also requested these. I'll double check with him during stand up.